### ISSUE TITLE
RabbitMQ connection reported as broken

### ISSUE THREAD

**Issue by zblesk** [wallabag/wallabag#3501]
### Issue details

There seem to be a bug in Wallabag's handling of RabbitMQ.
I've wasted much time, but haven't been able to get rid of the error on import page.

But when I went ahead and tried running `bin/console rabbitmq:consumer -e=prod import_firefox -w` it didn't complain, and I saw an active connection in RabbitMQ's dashboard. 🤔

Then I tried going to the firefox import page and importing a file - I just got a generic 'failed' message, logs don't show anything. Then I just tried again and it said 1 import queued.
When I ran the consumer console again, it seems the imports started.
So it isn't entirely broken, though the Rabbit error still gets reported, even though files are getting imported via rabbit.

### Environment

* wallabag version (or git revision) that exhibits the issue, according to `git log`: commit 79ad373d334d5352f9a18b2467c8e4ca9a549db9
Author: Nicolas Lœuillet <nicolas@loeuillet.org>
Date:   Mon Dec 11 19:56:14 2017 +0100
* How did you install wallabag? Via `git clone`
* Last wallabag version that did not exhibit the issue (if applicable): N/A
* php version: 7.0.22
* OS: ubuntu 16.04
* type of hosting (shared or dedicated): Dedicated VM in Azure cloud 
* which storage system you choose at install (SQLite, MySQL/MariaDB or PostgreSQL): SQLite

### Steps to reproduce/test case

As I mentioned in the gitter discussion where I'd been prompted to open this issue, I don't have much additional info. I couldn't get rid of the banner, even though the import process seems to be working (albeit very slowly.)

**Comment by j0k3r** [comment]
Few questions:
- does all queues are declared in Rabbit (ie: pocket, readability, wallabag_v1, wallabag_v2, firefox, chrome, instapaper, pinboard)?
- could comment line 37, 47, 48 and 49 [from that file](https://github.com/wallabag/wallabag/blob/master/src/Wallabag/ImportBundle/Controller/ImportController.php#L37-L49) and try again? You'll got an error while browsing to the import page. Share that exception here.

**Comment by zblesk** [comment]
No, the only queues in Rabbit are pocket and firefox, i.e. the ones I'd actually used. 

I commented out the try-catch as you suggested. Upon refresh, the error message disappeared, but no new one appeared. Everything looked OK.

**Comment by j0k3r** [comment]
Wow this is strange.
To check if RabbitMQ is working we are retrieving queue information for all queues, if it's fail we assume RabbitMQ isn't fully configured. Removing the try/catch you raise an error because you didn't defined all queues.
